lib/cli/index.js

```diff
@@ -25,10 +25,10 @@ function setup (options) {
 /**
  * Checks if there is a setup port or tries to find one.
  * @param  {Number}   port     Desired port
- * @return {Number|Promise}
+ * @return {Promise}
  */
 function getPort (port) {
-  return port || parseInt(process.env.PORT, 10) || findPort()
+  return port ? Promise.resolve(port) : findPort()
```

Comment:
Sorry! I didn't hit "submit review" this morning hahaha :P 

Maybe you can create a new branch and PR with the changes? (you'll be able to fix that commit message though 😝 )